index.html

```diff
@@ -1411,10 +1421,23 @@
             <li>Return <var>P</var>, but continue running these steps <a>in
             parallel</a>.
             </li>
-            <li>If the user agent is unable to <a>monitor the list of available
-            presentation displays</a> for the entire duration of the
-            <a>controlling browsing context</a> (e.g., because the user has
-            disabled this feature), then:
+            <li>If one of the following conditions is true, run the next
+            substeps:
+              <ul>
+                <li>The user agent is unable to <a>monitor the list of available
```

Comment:
Regarding resolving with `false` vs rejecting with `NotSupportedError`, I would suggest to deal with that in a follow-up because it's not related to this change.

However, IIRc, we did that so developers could take `NotSupportedError` as a clear signal that the discovery wasn't available but the feature would work otherwise.